src/representations/calculator_sorted_coulomb.hh

```diff
@@ -256,7 +256,7 @@ namespace rascal {
     //! if it is not a list of managers
     template <internal::CMSortAlgorithm AlgorithmType, class StructureManager,
               std::enable_if_t<
-                  not(internal::is_proper_iterator<StructureManager>::value),
+                  !(internal::is_proper_iterator<StructureManager>::value),
```

Comment:
Pity, I liked the "semantic" style (with `not`) much better. Are you sure there's no way of getting Doxygen to recognize that this is, in fact, a valid construct?

Reply:
The error does not come from doxygen but from breathe C++ parser (I guess used to separate function name and parameters when rendering doc)

Reply:
Oh. What happens if one puts a space before the `not`; does it try to parse it as a function then?

Reply:
This does not work either. However, I get other warnings from the parser even when replacing `not` by `!`, so I reverted the commit and this will have to be fixed later.

Reply:
Okay then. I did get the impression Breathe was a bit fragile in dealing with even slightly complicated syntax, but I don't see another solution so we'll just have to live with it for now.

Reply:
I don't think we should sacrifice code clarity to please the documentation... At least to me the ! for negation is quite bad to read. I am not suggesting to revert these changes though but these issues should probably be forwarded to the breathe developer so that they can think of a fix.

Reply:
> I am not suggesting to revert these changes

They are already reverted 😄